atomic_reactor/plugins/pre_inject_yum_repo.py

```diff
@@ -129,6 +129,9 @@ def run(self):
         """
         run the plugin
         """
+        if base_image_is_scratch(self.workflow.builder.base_image.to_str()):
+            self.log.debug("base image is scratch, do not inject yum repo")
+            return
```

Comment:
At the very end it runs /bin/rm -- but since the purpose is to build a usable base image, it seems likely this would be installed by the time the Dockerfile gets to the end.